### DISCUSSION TITLE
The new version 23.3.2 freezes from time to time

### DISCUSSION THREAD

**Discussion by boris-zl** [dbeaver/dbeaver#22377] [DBeaver Development]
The new version 23.3.2 freezes from time to time and need to be restarted.
Used on Ubuntu 22.04 (LTS)

**Comment by LonwoLonwo** [comment]
Could you please try to generate a thread dump and attach it here? Here is the instruction: https://dbeaver.com/docs/wiki/Making-a-thread-dump/
Thank you.
